src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -846,9 +848,9 @@ private async Task<string> CreatePullRequestAsync(List<UpdateAssetsParameters> u
         /// <param name="targetRepository">Target repository that the updates should be applied to</param>
         /// <param name="newBranchName">Target branch the updates should be to</param>
         /// <returns></returns>
-        private async Task CommitUpdatesAsync(
+        private async Task<string> CommitUpdatesAsync(
```

Comment:
I missed this change, and I don't agree with it. I don't think that method should do more than what it says on the name. Which is to commit the changes.

Reply:
But this method was calculating the PR description before, it just passed it to the caller via the StringBuilder

Reply:
Maybe we can change the naming a bit?

Reply:
Yep I think if it comes with a name change this makes some more sense

Reply:
I agree with Ricardo but it was also like this before. We were just sneaking it outside in a `StringBuilder` :) The simplest refactoring here would be to introduce some helper class `UpdateCommitResult` with one property called `PRDescription` and return this instead of bare string. It won't be a revolution but it would at least make explicit what we are returning.

Reply:
I'm totally fine with renaming this to CalculatePRDescriptionAndCommitUpdatesAsync to at least make it explicit. Of course bonus points if we can keep improving things as part of the refactoring, but I don't want to block on this since we were already doing it.